### PR TITLE
[DT-132][risk=no] adding hierarchy levels to SNOMED Procedure

### PR DESCRIPTION
**Pull request by freemabd**
adding hierarchy levels to SNOMED Procedure. Please reference https://precisionmedicineinitiative.atlassian.net/browse/DT-132

### CODE REVIEW

File: api/db-cdr/generate-cdr/build-prep-snomed-rel-meas.sh [hunk 1]
```diff
@@ -58,8 +58,8 @@ WHERE concept_id in
             and b.standard_concept = 'S'
     )"
 
-# currently, there are only 4 levels, but we run it 5 times to be safe
-for i in {1..4};
+# currently, there are only 6 levels, but we run it 7 times to be safe
+for i in {1..6};
```
Comment (chenchals) [inline api/db-cdr/generate-cdr/build-prep-snomed-rel-meas.sh:62]: Should this be 
for i in {1..7}; as the comment says? the comment and action are not consistent for this line

**Review by chenchals — CHANGES_REQUESTED** [review]

File: api/db-cdr/generate-cdr/build-prep-snomed-rel-meas.sh (hunk 1 above)
Comment (freemabd) [reply api/db-cdr/generate-cdr/build-prep-snomed-rel-meas.sh:62]: The for loop starts at 0, so this comment is correct but the others aren't. I will update the others to be consistent.

Comment (freemabd) [reply api/db-cdr/generate-cdr/build-prep-snomed-rel-meas.sh:62]: Good catch, fixing.

**Review by chenchals — APPROVED** [review]
Thank you

**Pull request merged**